config/imageService.js

```diff
@@ -54,6 +46,6 @@ module.exports.deleteImage = async key => {
 	} catch (err) {
 		console.log(err);
 		logger("error", "imageService", err);
-		throw err;
```

Comment:
Why is this remove? If we don't throw the error here then the try-catch surrounding deleteImg will not catch the error

Reply:
its not required as it should not block the update as it's server issue, not at client side, so if by chance, if an image is not deleted, we are not bothered, correct me if something wrong!

Reply:
Cul